docker/Dockerfile

```diff
@@ -1,4 +1,42 @@
-FROM debian:stretch
+FROM debian:stretch as builder
+
+ENV \
+    ARTIFACT_DIR=/tmp/artifacts \
+    DEBIAN_FRONTEND=noninteractive
+
+# Install tools for reading build dependencies; see `Build-Depends`
+# Squelch superfluous output; apt-get has poor verbosity control
+RUN \
+    apt-get update --quiet > /dev/null && \
+    apt-get install --quiet --quiet --no-install-recommends \
+        devscripts \
+        equivs \
+        git | grep "Setting up" | awk '{print $3 $4}' ORS=' '
+        # only print package names, versions
+
+WORKDIR /root/portal
```

Comment:
Probably a docker standard, but why `/root`?  I'd favor a POSIX structure...

Reply:
An odd quirk of the `root` user- their [`$HOME` is actually `/root`](http://www.linfo.org/slash_root.html)

I'll try using the `$HOME` env var (ie `$HOME/portal`), which would be a little more flexible (if we wanted to tighten up security and not run as root)

Reply:
That ended up working, nice improvement!